src/test/java/hello/HelloWorldTest.java

```diff
@@ -0,0 +1,39 @@
+package hello;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import com.bankofspring.hello.GreetingService;
+
+public class HelloWorldTest {
```

Comment:
Ah yes, I've thought about this. I thought I'd used different context files but must have ended up using the same one. Makes sense now.